cddl/concise-bom-tag.cddl

```diff
@@ -0,0 +1,6 @@
+concise-bom-tag = {
+  &(tag-identity: 0) => tag-identity-map
+  &(tags-list: 1) => [ + tag-identity-map ],
+  &(bom-validity: 2) => validity-map
+  ; TODO horizontal linking
```

Comment:
should add extension point here

Reply:
done in [25a1738](https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/41/commits/25a1738c05d62395e46c5715d472644d21c5a104)